core/src/main/kotlin/com/malinskiy/marathon/execution/device/DeviceActor.kt

```diff
@@ -15,22 +16,31 @@ import com.malinskiy.marathon.execution.withRetry
 import com.malinskiy.marathon.log.MarathonLogging
 import com.malinskiy.marathon.test.TestBatch
 import kotlinx.coroutines.CompletableDeferred
+import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.Job
 import kotlinx.coroutines.async
+import kotlinx.coroutines.cancelAndJoin
 import kotlinx.coroutines.channels.SendChannel
+import kotlinx.coroutines.delay
 import kotlinx.coroutines.isActive
 import kotlinx.coroutines.launch
 import kotlin.coroutines.CoroutineContext
 import kotlin.properties.Delegates
 
-class DeviceActor(private val devicePoolId: DevicePoolId,
-                  private val pool: SendChannel<DevicePoolMessage>,
-                  private val configuration: Configuration,
-                  val device: Device,
-                  private val progressReporter: ProgressReporter,
-                  parent: Job,
-                  context: CoroutineContext) :
-        Actor<DeviceEvent>(parent = parent, context = context) {
+class DeviceActor(
```

Comment:
FYI, when a branch includes a lot of indentation changes, that make meaningful review much harder. Perhaps, after this pull request, reindent the whole project in a new one for future convenience?

Reply:
I feel your pain, sorry. But I think it won't be the best move - if I will change a lot of files just for a single code style, it will be too hard to compare that branch with other branches, such as develop in main project or with your branch.